jekyll/_cci2/ssh-access-jobs.md

```diff
@@ -15,7 +15,7 @@ This document describes how to access a build container using SSH on CircleCI 2.
 ## Overview
 Often the best way to troubleshoot problems is to SSH into a job and inspect things like log files, running processes, and directory paths. CircleCI 2.0 gives you the option to access all jobs via SSH. Read our [blog post](https://circleci.com/blog/debugging-ci-cd-pipelines-with-ssh-access/) on debugging CI/CD pipelines with SSH.
 
-When you log in with SSH, you are running an interactive login shell. You are also likely to be running the command on top of the directory where the command failed the first time, so you are not starting a clean run. In contrast, CircleCI uses a non-interactive shell for running commands by default. Hence, steps run in interactive mode may succeed, while failing in non-interactive mode.
+When you log in with SSH, you are running an interactive login shell. You are may be running the command on top of the directory where the command failed the first time, **or** you may be running the command from the directory one level up from where the command failed (e.g. `~/project/` or `~/`). Either way, you will not be initiating a clean run. In contrast, CircleCI uses a non-interactive shell for running commands by default. Hence, steps run in interactive mode may succeed, while failing in non-interactive mode.
```

Comment:
```suggestion
When you log in with SSH, you are running an interactive login shell. You may be running the command on top of the directory where the command failed the first time, **or** you may be running the command from the directory one level up from where the command failed (e.g. `~/project/` or `~/`). Either way, you will not be initiating a clean run (you may wish to execute `pwd` or `ls` to ensure that you are in the correct directory).

Please note that a default CircleCI pipeline executes steps in a non-interactive shell and hence, there is the possibility that running steps using an interactive login may succeed, while failing in non-interactive mode.
```

Reply:
This is a bit of a tricky topic, and I had to read up on [different shell login types](https://askubuntu.com/questions/879364/differentiate-interactive-login-and-non-interactive-non-login-shell#:~:text=Reads%20commands%20from%20user%20input%20on%20a%20tty.&text=The%20user%20can%20interact%20with,don't%20require%20user%20inputs.) - so I tried to add a clarifying statement at the bottom.